lib_device_control/src/control.xc

```diff
@@ -1,11 +1,12 @@
-// Copyright 2016-2021 XMOS LIMITED.
+// Copyright 2016-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 #include <stdint.h>
 #include <stddef.h>
 #include <stdio.h>
 #include <xscope.h>
 #include "control.h"
-#include "control_transport.h"
+#include "control_shared.h"
```

Comment:
Control.h already includes control_shared?

Reply:
Good point. I will remove that include.